server/models/handlers/appointment.js

```diff
@@ -48,8 +48,39 @@ async function cancelWorkerAppointments(workerId, specificDate) {
     }
 }
 
+async function cancelSpecificAppointment(appointmentId) {
+    // If this method is somehow called without specifying values for the required parameters, false is returned.
+    if (_.isNil(appointmentId)) return false;
+    if (!_.isNumber(appointmentId)) return false; 
+
+    try {
+        //Check if appt exists
+        const doesAppointmentExist = await appointmentModel.appointmentExists(appointmentId);
+        if (doesAppointmentExist == false) return false;
+
+        //low priority, check if appt is not cancelled 
+
+        // Cancels specific appointment.
+        const cancelSpecificAppointment = await appointmentModel.cancelSpecificAppointment(appointmentId);
+        //if (_.isEmpty(cancelSpecificAppointment)) return false;
+        //if (_.isNil(cancelSpecificAppointment)) return "false cancelSpecificAppointment ";
+
+        //Finds corresponding workerTimeslotId 
+        const currentWorkerTimeslotId = await appointmentModel.findWorkerTimeslotId(appointmentId);
+        newStatus = 'available';
```

Comment:
Hi Peter,
newStatus does not look to be declared anywhere in this file yet so I would recommend initializing it with `const` or `var`. In addition, I would suggest leveraging the value in the constants file for greater flexibility.

Reply:
Thank you for the great suggestion, Amy! This has been completed.